### PR TITLE
Add payload-tracker announcer

### PR DESCRIPTION
**Pull request by SteveHNH**
Added a payload-tracker announcer and several instances where a message
will be sent to this announcer.

### CODE REVIEW

File: validators/types.go [hunk 1]
```diff
@@ -34,6 +34,19 @@ type Response struct {
...
+type Status struct {
+	Service     string `json:"service"`
+	Source      string `json:"source,omitempty"`
+	Account     string `json:"account"`
+	RequestID   string `json:"request_id"`
+	InventoryID string `json:"inventory_id"`
+	SystemID    string `json:"system_id"`
+	Status      string `json:"status"`
+	StatusMsg   string `json:"status_msg"`
+	Date        string `json:"date"`
```
Comment (jhjaggars) [inline validators/types.go]: You can use `time.Time` and forego the `Format` calls elsewhere.

Comment (SteveHNH) [reply validators/types.go]: does time.Time give you an isoformat though? I think the payload-tracker is dependent on the time we send to be the time it expects

Comment (jhjaggars) [reply validators/types.go]: The JsonMarshaller for `time.Time` does time.RFC3339 formatting for you.

https://golang.org/src/time/time.go?s=38016:38059#L1248

Comment (SteveHNH) [reply validators/types.go]: word. I'll get that dropped in

**Pull request merged**